src/views/Post/CreatePost.vue

```diff
@@ -44,33 +50,60 @@
         postData.value.shifts.splice(index, 1);
     };
 
-    function formatCurrency(event:any) {
-        const cleanedValue = event.target.value.replace(/\D/g, '');
-        const parsedValue = parseFloat(cleanedValue);
-        const formattedValue = parsedValue.toLocaleString('vi-VN', {
-            style: 'currency',
-            currency: 'VND',
-        });
-        postData.value.teachingFee = formattedValue;
+    function validateAgeRange() {
+        if (postData.value.requestTeacherAgeFrom && postData.value.requestTeacherAgeTo ) {
+            if (postData.value.requestTeacherAgeFrom >= postData.value.requestTeacherAgeTo) {
+                validationMessage.value = 'Tuổi bên trái phải nhỏ hơn Tuổi bên phải';
+                return false
+            } else {
+                validationMessage.value = '';
+                return true
+            }
+        } 
+        if (postData.value.requestTeacherAgeFrom || postData.value.requestTeacherAgeTo ) {
+            validationMessage.value = 'Tuổi bên trái và tuổi bên phải phải được chọn cả 2';
+                return false
+        }
+        return true
+    }
```

Comment:
Fix function validateAgeRange to this
```suggestion
    function validateAgeRange() {
        if (postData.value.requestTeacherAgeFrom || postData.value.requestTeacherAgeTo ) {
            validationMessage.value = 'Tuổi bên trái và tuổi bên phải phải được chọn cả 2';
                return false
        }
        if (postData.value.requestTeacherAgeFrom >= postData.value.requestTeacherAgeTo) {
            validationMessage.value = 'Tuổi bên trái phải nhỏ hơn Tuổi bên phải';
            return false
        } 
        validationMessage.value = '';
        return true
    }
```